setup

```diff
@@ -85,6 +83,7 @@ case "$1" in
 	hirteSetup "${ROOTFS}"
 	;;
     *)
+	systemctl stop qm.service 2>/dev/null || true
```

Comment:
Just a nit comment, as changing the line, probably good to add extra lines to verify the service is running and stop it.
 
```
if systemctl is-active --quiet "qm.service" ; then
  systemctl stop qm.service 2>/dev/null || true
fi
```

Reply:
pushed a commit addressing this comment. Dropped the trailing ` 2>/dev/null || true` as now we are testing if the service is active and if it fails to stop it's nice to know why.